scripts/get_github_dl_url.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+"""
+Extract the download URL for software hosted on github, taking into account OS and Architecture.
+"""
+import argparse
+import enum
+import platform
+import re
+import sys
+from typing import Callable, Iterable, NamedTuple
+
+import requests
+
+GITHUB_RELEASE_TEMPLATE = "https://api.github.com/repos/{}/releases/latest"
+
+OS = platform.system()
+ARCH = platform.machine()
+
+X86_64_ARCH_NAMES = {"x86_64", "arm64"}
```

Comment:
X86_64_ARCH_NAMES = {"x86_64", "amd64"}

Reply:
Good catch. I didn't run this on my linux box to test, I should have done that.

Reply:
Oh I _did_, my machine just says `x86_64`.